doc/intro.md

```diff
@@ -242,7 +237,7 @@ To solve this problem, Mimblewimble leverages another cryptographic concept (als
 coming from Confidential Transactions) called
 range proofs: a proof that a number falls within a given range, without revealing
 the number. We won't elaborate on the range proof, but you just need to know
-that for any `r*G + v*H` we can build a proof that will show that _v_ is greater than
+that for any `r*G + v*H` we can build a proof that shows that _v_ is greater than
 zero and does not overflow.
```

Comment:
I thought I remembered this being in the docs somewhere.
The rangeproof shows the value is `>=0` and we _do_ technically allow 0 value commitments.

Reply:
I missed this subtlety. It is indeed mentioned in the intro, but somewhere else. I checked and couldn't find this mistake somewhere else.